packages/package/.gitignore

```diff
@@ -6,6 +6,8 @@
 !/test/node_modules
 /test/apps/basics/test/errors.json
 .custom-out-dir
+test/watch/src/lib/*
+!test/watch/src/lib/index.js
```

Comment:
why are these lines needed? I don't see "watch" anywhere else in this diff

also, I notice the other lines have a leading `/` whereas this one doesn't

Reply:
The package test creates some temporary files that aren't cleaned up if the test gets interrupted.

I'll change the slash.